### PR TITLE
AIs and cyborgs show up on the OOC crew manifest

### PR DESCRIPTION
**Pull request by DamianX**
:cl:
 * tweak: AIs and cyborgs are now displayed in the OOC (for people at the lobby) crew manifest. In-game crew manifest is unaffected.

### CODE REVIEW

**Comment by PaulRitter** [comment]
`var/mob/living/silicon/ai/dooropener`

i approve this pr

**Comment by ValkyrieSkies** [comment]
THANK YOU.

File: code/defines/obj.dm [hunk 1]
```diff
@@ -99,6 +99,17 @@
...
+	if(OOC)
+		for(var/mob/living/silicon/ai/dooropener in mob_list)
+			bot[dooropener.name] = "AI"
+			isactive[dooropener.name] = (dooropener.client?.inactivity > 10 * 60 * 10) ? "Active" : "Inactive"
+		for(var/mob/living/silicon/robot/tincan in mob_list)
+			if(isMoMMI(tincan))
+				continue
+			if(tincan.syndicate)
+				continue
+			bot[tincan.name] = "Cyborg"
```
Comment (ValkyrieSkies) [inline code/defines/obj.dm]: Isn't 'Android' used when borgs are created with Posibrains? Might want to catch that one too.

**Comment by SonixApache** [comment]
someone finally did it

**Pull request merged**